test/RelayServer.webpack.test.js

```diff
@@ -0,0 +1,27 @@
+// eslint-disable-next-line no-unused-vars
```

Comment:
Please use TypeScript

Reply:
Please use TypeScript for files you create. I don't want to add a single line of JavaScript to the project.